src/Tools/SarifValidator/Options.cs

```diff
@@ -20,5 +20,12 @@ internal class Options
             HelpText = "Path of the SARIF file to validate",
             Required = true)]
         public string InstanceFilePath { get; set; }
+
+        [Option(
+            'l',
+            "log-file-path",
```

Comment:
I would prefer that this tool follow the driver framework conventions, which use OutputFilePath as an argument, take a look at AnalyzeCommandBase. You can propose changes to that if you'd like, but binskim, sqlskim, this tool should all conform (in the interests of building an eco-system with consistent command-line interface)